projects/1.20-fabric/assets/debugify/debugify/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+{
+	"debugify.name": "Debugify",
+	"debugify.basic": "基础修复",
+	"debugify.gameplay": "游戏修复",
+	"debugify.gameplay.warning": "警告：此类别包含可能对其他人来说不公平的修复！",
+	"debugify.gameplay.enable_in_multiplayer": "在多人游戏中启用",
+	"debugify.fix.enabled": "已修复",
+	"debugify.fix.disabled": "未修复",
+	"debugify.fix.unavailable": "不可用",
+	"debugify.error.conflict": "‘%s’修复与模组‘%s’不兼容",
+	"debugify.error.os": "%s仅适用于%s",
+	"debugify.error.mixin_error": "加载%s失败，有关详细信息，请查看日志。",
+	"debugify.error.mixin_error.text": "错误",
+	"debugify.os.windows": "Windows",
+	"debugify.os.macos": "macOS",
+	"debugify.os.linux": "Linux",
+	"debugify.os.solaris": "Solaris",
+	"debugify.misc": "杂项",
+	"debugify.misc.default_disabled": "默认为禁用",
+	"debugify.misc.default_disabled.description": "默认禁用新的错误修复，而不是启用。",
+	"debugify.env.client": "客户端",
+	"debugify.env.client.desc": "只适用于客户端的修复。",
+	"debugify.env.server": "主要",
+	"debugify.env.server.desc": "适用于客户端和服务器的修复，以及它们之间的共同代码。",
+	"debugify.no_yacl.title": "未安装 YetAnotherConfigLib！",
+	"debugify.no_yacl.description": "要使用 GUI 配置 Debugify，你必须安装%s，这是用来实现这一功能的前置库。否则，你可以使用存储在「%s」的JSON文件进行配置。",
+
+	"debugify.mc_237493.header": "遥测已由 Debugify 关闭",
+	"debugify.mc_237493.line": "你在遥测选项上选择了「%s」，所以 Debugify 正在阻止与 Mojang 的遥测服务的任何连接。这是由 Debugify 提供的MC-237493修复。",
+	"debugify.mc_237493.tooltip.off": "「%s」完全禁用了与 Mojang 遥测服务的任何连接。（Debugify MC-237493）",
+	"debugify.mc_237493.tooltip.minimal": "「%s」只包括 Mojang 认为需要的信息。",
+	"debugify.mc_237493.tooltip.all": "「%s」包括所有的最少和可选遥测。",
+
+	"debugify.fix_explanation.mc-55347": "断开连接后清除屏幕标题。",
+	"debugify.fix_explanation.mc-79545": "在渲染时将经验等级限制为整型限制。",
+	"debugify.fix_explanation.mc-93384": "将溺水粒子生成位置提升到眼睛高度。",
+	"debugify.fix_explanation.mc-108948": "问题是客户端运行的计算逻辑与服务器不同。当没有人控制船时，客户端不会在运行船的物理模拟，导致服务端与客户端不同步。这个修复只是告诉客户端运行船的物理计算。",
+	"debugify.fix_explanation.mc-111516": "修改数学acos计算以确保结果不超过1.0",
+	"debugify.fix_explanation.mc-112730": "如果信标和其他方块实体已添加到全局列表中，则会阻止它们添加到本地方块实体渲染器列表中。",
+	"debugify.fix_explanation.mc-122477": "防止在编辑栏（例如聊天栏）初始化后进行前两次键盘轮询。",
+	"debugify.fix_explanation.mc-123739": "按标识符对配方条目进行排序。",
+	"debugify.fix_explanation.mc-143474": "重生后生成新的本地玩家时，复制选定的物品槽。",
+	"debugify.fix_explanation.mc-159163": "从实体数据包处理中删除姿势数据。",
+	"debugify.fix_effect.mc-159163": "这将阻止客户端与服务器同步姿势数据，完全由客户端来决定姿势。",
+	"debugify.fix_explanation.mc-162253": "在加载区块时停止客户端光照更新。",
+	"debugify.fix_explanation.mc-165381": "当客户端丢弃物品时取消破坏方块",
+	"debugify.fix_explanation.mc_176559": "在检查是否应重置破坏方块时与比较物品变化时不考虑物品耐久。",
+	"debugify.fix_explanation.mc-197260": "覆盖传递给盔甲架渲染器的光照等级，最大值为：盔甲架下方的方块、盔甲架底部、盔甲架顶部和盔甲架上方的方块。",
+	"debugify.fix_explanation.mc-237493": "在遥测菜单中添加一个选项以完全禁用遥测。通过将所有传出的遥测事件覆盖为未发送的空遥测事件，可以禁用遥测。",
+
+	"debugify.fix_explanation.mc-2025": "由于浮点数不准确，有时实体的判定箱会比期望的稍小。然后，如果在实体相互挤压之前发生这种情况，它们将穿进墙里。然后，当在方块负载上重新计算AABB时，它们将被确定为墙内，然后被推到墙内。此修复程序只需将“AABB”标记写入具有双精度大小的判定箱实体NBT数据，然后将其重新加载。",
```

```suggestion
	"debugify.fix_explanation.mc-2025": "由于浮点数不准确，有时实体的判定箱会比期望的稍小。然后，如果在实体相互挤压之前发生这种情况，它们将穿进墙里。然后，当在方块负载上重新计算AABB时，它们将被确定为墙内，然后被推到墙内。此修复仅将“AABB”标记写入具有双精度大小的判定箱实体NBT数据，然后将其重新加载。",
```
这一整段的前面部分读起来很怪（复用`然后`，`被确定为墙内`）
对AABB不熟，但是还是建议改一下